tests/targetAcquisition_geolocation_test.py

```diff
@@ -54,19 +45,20 @@ def test_targetAcquisition_to_geolocation(get_image):
 
     target.set_curr_frame(merged)
     check1, coordinates_and_telemetry = target.get_coordinates()
+    print(check1,coordinates_and_telemetry)
 
     location.set_constants()
-    check2, geo_coordinates = location.run_locator(merged.telemetry, [[0, 0],[60, 523], [200,0], [430,505]])
-    # check3, locations = location.run_output(geo_coordinates) 
-    #print (check1, coordinates_and_telemetry)
+    check2, geo_coordinates = location.run_locator(target.telemetryData, [[0, 0],[60, 523], [200,0], [430,505]])
```

Comment:
Should the input not be from `coordinates_and_telemetry` ? Not the class object itself?